keps/sig-instrumentation/2831-kubelet-tracing/kep.yaml

```diff
@@ -0,0 +1,33 @@
+title: Kubelet OpenTelemetry Tracing
+kep-number: 2831
+authors:
+  - "@husky-parul"
+  - "@somalley"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-architecture
+  - sig-node
```

Comment:
SIG Node acked the KEP at the [2022-01-25 meeting](https://docs.google.com/document/d/1Ne57gvidMEWXR70OxxnRkYquAoMpt56o75oZtg-OeBg/edit#bookmark=id.spr2uizb8qi4).